src/test/java/com/wikia/webdriver/common/core/EmailUtils.java

```diff
@@ -52,6 +52,7 @@ public static String getFirstEmailContent(String userName, String password, Stri
 
       for (int i = 0; !forgottenPasswordMessageFound; i++) {
         messages = inbox.getMessages();
+        Thread.sleep(2000);
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this use of "Thread.sleep()". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=squid%3AS2925)

[project-id]: com.wikia.selenium-tests